src/__tests__/Table.spec.js

```diff
@@ -1,83 +1,71 @@
-    expect(wrapper.hasClass('table')).toBe(true);
-    expect(wrapper.find('div').length).toBe(1);
+    testForCustomTag(Table);
   });
 
   it('should render modifier classes', () => {
-    const wrapper = shallow(
-      <Table size="sm" bordered striped dark hover>
-        Yo!
-      </Table>,
-    );
-
-    expect(wrapper.text()).toBe('Yo!');
-    expect(wrapper.hasClass('table')).toBe(true);
-    expect(wrapper.hasClass('table-sm')).toBe(true);
-    expect(wrapper.hasClass('table-bordered')).toBe(true);
-    expect(wrapper.hasClass('table-striped')).toBe(true);
-    expect(wrapper.hasClass('table-hover')).toBe(true);
-    expect(wrapper.hasClass('table-dark')).toBe(true);
+    render(<Table data-testid="table" size="sm" bordered striped dark hover />);
+    const node = screen.getByTestId('table');
+    expect(node).toHaveClass('table');
+    expect(node).toHaveClass('table-sm');
+    expect(node).toHaveClass('table-bordered');
+    expect(node).toHaveClass('table-striped');
+    expect(node).toHaveClass('table-hover');
+    expect(node).toHaveClass('table-dark');
   });
 
   it('should render a borderless table', () => {
-    const wrapper = shallow(<Table borderless>Yo!</Table>);
-
-    expect(wrapper.text()).toBe('Yo!');
-    expect(wrapper.hasClass('table')).toBe(true);
-    expect(wrapper.hasClass('table-borderless')).toBe(true);
+    render(<Table data-testid="table" borderless />);
+    expect(screen.getByTestId('table')).toHaveClass('table');
+    expect(screen.getByTestId('table')).toHaveClass('table-borderless');
   });
 
   it('should render responsive wrapper class', () => {
-    const wrapper = shallow(<Table responsive>Yo!</Table>);
-
-    expect(wrapper.text()).toBe('Yo!');
-    expect(wrapper.hasClass('table-responsive')).toBe(true);
-    expect(wrapper.find('.table').length).toBe(1);
+    render(<Table data-testid="table" responsive />);
+    expect(screen.getByTestId('table')).toHaveClass('table');
+    expect(screen.getByTestId('table').parentNode).toHaveClass(
+      'table-responsive',
+    );
   });
 
   it('should render responsive wrapper class for md', () => {
-    const wrapper = shallow(<Table responsive="md">Yo!</Table>);
-
-    expect(wrapper.text()).toBe('Yo!');
-    expect(wrapper.hasClass('table-responsive-md')).toBe(true);
-    expect(wrapper.find('.table').length).toBe(1);
+    render(<Table data-testid="table" responsive="md" />);
+    expect(screen.getByTestId('table')).toHaveClass('table');
+    expect(screen.getByTestId('table').parentNode).toHaveClass(
+      'table-responsive-md',
+    );
   });
 
   it('should render responsive wrapper cssModule', () => {
     const cssModule = {
       table: 'scopedTable',
       'table-responsive': 'scopedResponsive',
     };
-    const wrapper = shallow(
-      <Table responsive cssModule={cssModule}>
-        Yo!
-      </Table>,
+    const { debug } = render(
+      <Table data-testid="table" responsive cssModule={cssModule} />,
+    );
+    debug();
```

Comment:
Thank you for your contribution!

Can you remove this `debug()` statement?

Reply:
Looks like this was resolved 👍